experiment/runner.py

```diff
@@ -26,6 +27,9 @@
 import threading
 import time
 import zipfile
+from random import Random
```

Comment:
Google's style is to only import modules. So please change this to `import random` and then use `random.Random`.

Please do this for numpy as well, and please keep imports in alphabetical order.

Reply:
Should be all set on this, but I did need to modify the `.pylintrc` file to workaround this issue:

https://github.com/pylint-dev/pylint/issues/1975